src/components/form/form-question.tsx

```diff
@@ -41,8 +43,8 @@ export function FormQuestion({ column }: { column: FormColumn }) {
       ) : null}
       {UIDataType[column.uidt] === UIDataType.Checkbox ? (
         <div className='flex w-full flex-col items-start justify-start'>
-          <label className='relative flex w-full cursor-pointer select-none flex-row-reverse items-center justify-end'>
-            <p className='ml-2 text-black'>ยอมรับ</p>
+          <span className='relative flex w-full cursor-pointer select-none flex-row-reverse items-center justify-end'>
```

Comment:
Please do not use `span` as wrapper as it might cause hydration error.
My suggestion is to simply use `div`.